workloads/aws-oidc/aws-oidc-consumer/deploy.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: production
```

Comment:
I guess in this demo case we don't need the `analytics` namespace? More just a question for my understanding!